cluster-scope/overlays/moc/groups/thoth.enc.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+    name: thoth
+    annotations:
+        kustomize.config.k8s.io/behavior: replace
+users:
+- ENC[AES256_GCM,data:LYosGvpof2CiUW9WwLZlSA==,iv:Qr7bX+QA+viqI2BJaoukJJhCyXwhXowfJTJelhAt63M=,tag:DHARXG7CdBdsj5QCfxn5Rg==,type:str]
+- ENC[AES256_GCM,data:Ls+fuG5HFyXxAZ8fNECQX0k=,iv:q7kpyVNDS7dPVMNc9HBVTdidQmLo+RDOor4gvl/iR9w=,tag:7DcURkZuz5FHbGaIqP55Hg==,type:str]
```

Comment:
Christoph's email here is different from what's on live, and what was introduced [here](https://github.com/operate-first/apps/blob/master/namespaces/overlays/moc/thoth-amun-api-prod/group.enc.yaml#L6).

Reply:
My content was taken from the MOC cluster before the Harshad's PRs. But Harshad's email is the correct one. Fixing it now.